core/src/main/resources/jenkins/model/Jenkins/oops.jelly

```diff
@@ -32,7 +32,7 @@ THE SOFTWARE.
     <l:header />
     <l:main-panel>
         <h1 style="text-align: center">
-          <img src="${imagesURL}/rage.svg" height="179" width="154"/> <span style="font-size:50px"><st:nbsp/>${%Oops!}</span>
+          <img src="${imagesURL}/rage.svg" height="179" width="154" alt="null"/> <span style="font-size:50px"><st:nbsp/>${%Oops!}</span>
```

Comment:
```suggestion
          <img src="${imagesURL}/rage.svg" height="179" width="154" alt="Oops"/> <span style="font-size:50px"><st:nbsp/>${%Oops!}</span>
```

Reply:
Decorative only. Also, `null` isn't intended as a literal value, but to represent the explicitly specified empty value (`alt=""`).
```suggestion
          <img src="${imagesURL}/rage.svg" height="179" width="154" alt=""/> <span style="font-size:50px"><st:nbsp/>${%Oops!}</span>
```

Reply:
Resolved in 4d8fd426116664abdb7d45af29ba2e226821b76d